cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/RefResolver.java

```diff
@@ -42,10 +45,10 @@ private RefResolver(Object yamlTreeRoot) {
      * @return the new root of the specified yaml tree after the ref-resolving process
      * @throws RefResolvingException if an error during the ref-resolution process occurs
      */
-    public static Object resolveRefs(Object yamlTreeRoot) {
+    public static Object resolveRefs(Object yamlTreeRoot, String rootFilePath) {
```

Comment:
The additional parameter should be documented above

Reply:
True enough. Let me quickly document the parameter.